tests/sles4sap/publiccloud/qesap_ansible.pm

```diff
@@ -54,6 +54,19 @@ sub run {
             '/tmp/results/',
             '-type', 'f',
             '-iname', "*.xml");
+        my $find_ansible_know_issue = qr/\[OSADO\]\[softfail\] bsc#(\d+) (.*)/;
+        my $cat_command = "cat '$ret[1]'";
+        my $ansibleOutput = script_output($cat_command);
```

Comment:
I think you can safely combine these 2 lines as there is no other use of `$cat_command` later.

Reply:
ok